lab-david/lib/error-middleware.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const createError = require('http-errors');
+const debug = require('debug')('cfgram:error-middleware');
+
+module.exports = function(req, res, next){
+  debug('error middleware');
+
+  console.error('message:', err.message);
+  console.error('name:', err.name);
+
+  if(err.status){
+    res.status(err.status).send(err.name);
+    next();
+    return;
+  }
+
+  if(err.name === 'ValidationError'){
```

Comment:
One of many possible mongo errors. I encourage you to find more errors and account for them so you don't have to just send back a 500.